corehq/apps/style/static/preview_app/less/preview_app/form.less

```diff
@@ -40,12 +40,7 @@
 }
 
 .form-single-question {
-  padding-top: 66px;
-  min-height: 443px;
-}
-
-.preview-tablet-mode .form-single-question {
-  min-height: 642px;
```

Comment:
@biyeun it's kind of hard to tell what these numbers are supposed to be representing. i think the styles look fine now, but would be good if you took a look or could point me to why those are needed